### PR TITLE
[NGMIMEType stringEncodingForCharset:] added more ISO charsets

### PR DESCRIPTION
**Pull request by javierag**
I have added some more encodings for iso charsets. I have no setup a mail client to generate them but in anycase they will be better that the latin-2 which we were using.

### CODE REVIEW

**Comment by javierag** [comment]
What to test:
There are several fixes here and changes that affect the RTF parsing. However I think we should manually test the errors that motivated this work:

- test on the notes/description field of contacts and events
- test with english text
- test with text with does not use a latin alphabet
- test with text which contains ñáéíóúÁÉÍÓÚ

Remember that the parsing is not complete so an error could  be an previous existent error and in that case I should be left for later.

File: sope-mime/NGMime/NGMimeType.m [hunk 1]
```diff
@@ -88,6 +88,7 @@ + (void)initialize {
   }
 }
 
+// TODO: this should be reimplemented with a lookup table
```
Comment (sixstone-qq) [inline sope-mime/NGMime/NGMimeType.m]: It would be great to uniform the comments to use /* */ or //, I'd prefer the former...

**Comment by blaxter** [comment]
rebase needed

**Comment by javierag** [comment]
This is already in master..

**Pull request closed without merging**